packages/dds/tree/src/domains/json/jsonDomainSchema.ts

```diff
@@ -28,19 +29,23 @@ const jsonPrimitives = [...leaf.primitives, leaf.null] as const;
 /**
  * Types allowed as roots of Json content.
  */
-export const jsonRoot = [() => jsonObject, () => jsonArray, ...jsonPrimitives] as const;
+export const jsonRoot: FlexAllowedTypes = [
+	(): FlexTreeNodeSchema => jsonObject,
```

Comment:
do these need lambda syntax anymore?

Reply:
yes, thats to fix the runtime issue of using things before they are defined, and has nothing to do with the typing

Reply:
oh, I missed the objects they refer to are exported. was thinking we could inline them. nvm!